doc/guides/4 - Refinery Extensions/3 - Testing.textile

```diff
@@ -26,6 +26,12 @@ And run
 $ bundle install
 </ruby>
 
+You will need to install phantomjs from http://phantomjs.org/ or through your OS package manager.
+
+<shell>
+$ sudo apt-get install phantomjs
+</shell>
+
```

Comment:
Nice.. could you document that apt-get is for Linux and add these other examples please:

```
OS X with homebrew:
<shell>
$ brew install phantomjs
</shell>

With npm:
<shell>
$ npm install -g phantomjs
</shell>
```